command/report/report.go

```diff
@@ -256,20 +252,14 @@ func (opts *ReportOptions) Run() int {
 	// }
 
 	if !queryResponse.Data.CreateArtifact.Ok {
-		fmt.Println("DeepSource | Error | Reporting failed | ", queryResponse.Data.CreateArtifact.Error)
+		fmt.Fprintln(os.Stderr, "DeepSource | Error | Reporting failed |", queryResponse.Data.CreateArtifact.Error)
 		sentry.CaptureException(errors.New(queryResponse.Data.CreateArtifact.Error))
 		return 0
```

Comment:
```suggestion
		return 1
```

Reply:
@subham-deepsource there are some other instances above where we return `0` here. Ideally we should return `1` in those failure cases. Can you please make that change?

Reply:
Yeah, right.